Orange/widgets/data/tests/test_owfile.py

```diff
@@ -176,6 +176,19 @@ def test_file_not_found(self):
         self.open_dataset("iris")
         self.assertFalse(self.widget.Error.file_not_found.is_shown())
 
+   def test_nothing_selected(self):
```

Comment:
Tests don't pass because the indent is too small by one space. (Trick: first run tests on your machine. Not that I'm without a sin in this respect, though. :))